### ISSUE TITLE
HEPA Air Filter/Purifier

### ISSUE THREAD

**Issue by nelsonic** [dwyl/home#309]
There was recently a forest fire nearby @home and the air was "smoky" for a few days ... 🚬 
Also, we live near a road and there is inevitably some air-quality downside. 🚗 😕 
We have an `IKEA` air filter in the workshop downstairs and it works reasonably well 👌 
https://www.ikea.com/pt/en/p/foernuftig-air-purifier-white-50461937/
<img width="1392" alt="image" src="https://github.com/user-attachments/assets/b6d47eb1-bdde-49ea-b9a2-4c697df4f918">

The filters aren't cheap and they were out-of-stock when we tried to buy a few weeks ago, 
now back in-stock: (so I will be getting a couple next time we are @IKEA...) 
https://www.ikea.com/pt/en/p/foernuftig-2-piece-filter-set-s19444196/
<img width="1392" alt="image" src="https://github.com/user-attachments/assets/f5b824b0-aaf7-4792-84ee-9188d0af3ee3">

But the _most_ important thing is the _noise_ level and energy consumption. 🙉 
When the `IKEA` filter is on lowest level you can still hear the hum ... and it consumes `14W`.
The `IKEA` filter has a "crude" (but functional) physical dial, perfect for the workshop.

This `Philips` 600 series Air Purifier is 10db quieter and uses `12W` of energy.

https://www.amazon.es/-/pt/gp/product/B0CCF2NKCW
<img width="1354" alt="image" src="https://github.com/user-attachments/assets/ccc465e2-635f-4740-923d-637a413efae1">

https://www.philips.co.uk/c-p/AC0651_10/600-series-air-purifier
<img width="1354" alt="image" src="https://github.com/user-attachments/assets/92a367f9-252c-4e81-a825-cf120eb41975">

<img width="1008" alt="image" src="https://github.com/user-attachments/assets/8e78a811-5ff3-4985-9724-0193b91d4cd7">

<img width="987" alt="image" src="https://github.com/user-attachments/assets/6bbf5b46-e284-4713-b5a3-5a84685e3c24">

The filters cost `€25` and should be replaced every 12 months:
https://www.amazon.es/-/pt/dp/B0BX741G4P

<img width="1356" alt="image" src="https://github.com/user-attachments/assets/f5f73e80-2882-4b81-82bc-5444634ef699">

Though I'm _much_ cheaper from Ali Express: https://pt.aliexpress.com/w/wholesale-AC0650.html
<img width="896" alt="image" src="https://github.com/user-attachments/assets/27ec2e51-8590-46a7-9244-802f0d507eb4">

e.g: https://pt.aliexpress.com/item/1005007636139108.html **`€16/each`** with free shipping. 📦 ⏳ 

So to summarise: Lower filter costs, less noise and energy consumption than the `IKEA` filter. 

@iteles thoughts? if you have already researched this, please share. 🙏

**Comment by nelsonic** [comment]
@iteles BTW: The biz price for the Philips filter is **`€ 66,11`** which I think is _very_ reasonable. 
<img width="948" alt="image" src="https://github.com/user-attachments/assets/1621b89c-c4e6-44de-85fe-aafedbb7fd99">

**Comment by iteles** [comment]
You bought this before I saw the issue. Sounds good.

**Comment by nelsonic** [comment]
@iteles, yes, I placed an order that is, as you know from experience, _very easy_ to `return` unopened. 📦 
To avoid the price increasing again like on the vac battery ... #302 🙃 
Thanks for looking at this. 👌

**Comment by nelsonic** [comment]
@iteles where did this end up? 🤷‍♂️

**Comment by iteles** [comment]
Meire moved it from the bottom of the stairs to under the stairs next to the Xmas boxes.

**Comment by iteles** [comment]
It is still there. 
There have been a lot of queimas recently so it may require use if it stops raining again.

**Comment by nelsonic** [comment]
@iteles where would you like to run it, would you be comfortable to have it running in our bedroom? 💭
It's meant to be ultra quiet and no light. 🤞

**Comment by iteles** [comment]
Our bedroom in the very short term to see what it says and then playroom for now.

**Comment by nelsonic** [comment]
Please unbox it and set it up on _your_ phone. 🙏
